Cargo.toml

```diff
@@ -189,6 +85,130 @@ feat_common_core = [
+feat_os_macos = [
+  "feat_os_unix", ## == a modern/usual *nix platform
+]
+# "feat_os_unix" == set of utilities which can be built/run on modern/usual *nix platforms
+feat_os_unix = [
+  "feat_Tier1",
+  #
+  "feat_require_crate_cpp",
+  "feat_require_unix",
+  "feat_require_unix_utmpx",
+]
+# "feat_os_windows" == set of utilities which can be built/run on modern/usual windows platforms
+feat_os_windows = [
+  "feat_Tier1", ## == "feat_os_windows_legacy" + "hostname"
+  ]
+## (secondary platforms) feature sets
+# "feat_os_unix_gnueabihf" == set of utilities which can be built/run on the "arm-unknown-linux-gnueabihf" target (ARMv6 Linux [hardfloat])
+feat_os_unix_gnueabihf = [
+  "feat_Tier1",
+  #
+  "feat_require_unix",
+  "feat_require_unix_utmpx",
+]
+# "feat_os_unix_musl" == set of utilities which can be built/run on targets binding to the "musl" library (ref: <https://musl.libc.org/about.html>)
+feat_os_unix_musl = [
+  "feat_Tier1",
+  #
+  "feat_require_unix",
+]
+## feature sets with requirements (restricting cross-platform availability)
+#
+# ** NOTE: these `feat_require_...` sets should be minimized as much as possible to encourage cross-platform availability of utilities
+#
+# "feat_require_crate_cpp" == set of utilities requiring the `cpp` crate (which fail to compile on several platforms; as of 2020-04-23)
+feat_require_crate_cpp = [
+  "stdbuf",
+]
+# "feat_require_unix" == set of utilities requiring support which is only available on unix platforms (as of 2020-04-23)
+feat_require_unix = [
+  "chgrp",
+  "chmod",
+  "chown",
+  "chroot",
+  "du",
+  "groups",
+  "hostid",
+  "id",
+  "install",
+  "kill",
+  "logname",
+  "mkfifo",
+  "mknod",
+  "nice",
+  "numfmt",
+  "nohup",
+  "pathchk",
+  "stat",
+  "timeout",
+  "tty",
+  "uname",
+  "unlink",
+]
+# "feat_require_unix_utmpx" == set of utilites requiring unix utmp/utmpx support
+# * ref: <https://wiki.musl-libc.org/faq.html#Q:-Why-is-the-utmp/wtmp-functionality-only-implemented-as-stubs?>
+feat_require_unix_utmpx = [
+  "pinky",
+  "uptime",
+  "users",
+  "who",
+]
+## (alternate/newer/smaller platforms) feature sets
+# "feat_os_unix_fuchsia" == set of utilities which can be built/run on the "Fuschia" OS (refs: <https://fuchsia.dev>; <https://en.wikipedia.org/wiki/Google_Fuchsia>)
+feat_os_unix_fuchsia = [
+  "feat_common_core",
+  #
+  "feat_require_crate_cpp",
+  #
+  "chgrp",
+  "chmod",
+  "chown",
+  "du",
+  "groups",
+  "hostid",
+  "install",
+  "logname",
+  "mkfifo",
+  "mknod",
+  "nice",
+  "pathchk",
+  "tty",
+  "uname",
+  "unlink",
+]
+# "feat_os_unix_redox" == set of utilities which can be built/run on "Redox OS" (refs: <https://www.redox-os.org>; <https://en.wikipedia.org/wiki/Redox_(operating_system)>)
+feat_os_unix_redox = [
+  "feat_common_core",
+  #
+  "uname",
+  "chmod",
+  "install",
+]
+# "feat_os_windows_legacy" == slightly restricted set of utilities which can be built/run on early windows platforms (eg, "WinXP")
```

Comment:
Do we care about this?

Reply:
I still do.
It's still useful for non-Win10 windows platforms, of which there are still a huge number.